Makefile

```diff
@@ -67,7 +67,9 @@ endif
 
 .PHONY: lint
 lint: ## Run linter over the codebase
-	time "$(GOBIN)/gometalinter" ./pkg/... ./cmd/... ./integration/...
+	#  Install golangci-lint here till the build docker image is updated
+	go get github.com/golangci/golangci-lint/cmd/golangci-lint
```

Comment:
Please remove this line. It should just be part of the install-build-deps.sh` file like we do with other dependencies.

Reply:
I tried but as mentioned in `install-build-deps.sh`, I need to run `make -f Makefile.docker update-build-image-manifest && make -f Makefile.docker push-build-image`. However, I don't have the permission to push the image. https://github.com/weaveworks/eksctl/pull/1443#discussion_r334655873

Hence, that is why I added above comment along with `go get` as workaround

Reply:
I'm not sure since we have been changing the build pipeline lately but it should be able to build the image anyway without failing. After it's merged, it will push the new image (otherwise I can push it manually).

Reply:
Sure, let me know if you have another comment, happy to address :)

Reply:
Can you remove the `go get ...`? Let's see what the CI does about it...

Reply:
Without go get https://circleci.com/gh/weaveworks/eksctl/5560?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link
```
time "/go/bin/golangci-lint" run
time: can't execute '/go/bin/golangci-lint': No such file or directory
```

Reply:
@sayboras I think we are ready to get this through :) Can you please remove this line and add `github.com/golangci/golangci-lint/cmd/golangci-lint` to the  `.requirements` file?

Reply:
sure :)

Reply:
@martina-if let me know if anything else is required.